.circleci/config.yml

```diff
@@ -2141,7 +2137,10 @@ workflows:
       - cannon-prestate:
           requires:
             - go-mod-download
-      - contracts-bedrock-build
+      - contracts-bedrock-build:
+          build_command: forge build --skip test
+          context:
+            - slack
```

Comment:
Seems like we should be verifying the contracts build before the PR can merge so this shouldn't ever fail.  Having to specify custom build commands feels like it's adding duplication and is going to get out of sync so we aren't necessarily testing contracts built the same was as we would for a release here. Might be more maintainable for `contracts-bedrock-build` to just accept a `skip_tests` param and it always controls the build command.

Reply:
Agreed, am working to get there.

Reply:
Recent foundry feature that adds junit test report files will make this possible